src/test/java/core/basesyntax/service/FileReaderImplTest.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax.service;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FileReaderImplTest {
+    private static FileReader fileReader;
+    private static final String FILE_PATH = "src/main/resources/fruitbase.csv";
```

Comment:
maybe you could create a resource folder in the test folder, 
then you can create two files there and test the tests with those files